microsoft-365/security/defender-endpoint/linux-preferences.md

```diff
@@ -726,6 +747,46 @@ The following configuration profile contains entries for all settings described
       "automaticSampleSubmissionConsent":"safe",
       "automaticDefinitionUpdateEnabled":true,
       "proxy": "<EXAMPLE DO NOT USE> http://proxy.server:port/"
+   },
+   "exclusionSettings":{
+        "exclusions":[
+           {
+              "$type":"excludedPath",
+              "isDirectory":true,
+              "path":"/home/*/git<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "epp"
+              ]
+           },
+           {
+              "$type":"excludedPath",
+              "isDirectory":true,
+              "path":"/run<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "global"
+              ]
+           },
+           {
+              "$type":"excludedPath",
+              "isDirectory":false,
+              "path":"/var/log/system.log<EXAMPLE DO NOT USE><SAME AS APPLYING ONLY GLOBAL>",
+              "scopes": [
+                    "epp", "global"
+              ]
+           },
+           {
+              "$type":"excludedFileExtension",
+              "extension":".pdf<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "epp"
+              ]
+           },
+           {
+              "$type":"excludedFileName",
```

Comment:
We have the time to change the name of this to excludedProcessName to make it more apparent.

Reply:
Could have been done in the same release, now this can be taken up in future.